types/node/buffer.d.ts

```diff
@@ -113,7 +113,7 @@ declare module 'buffer' {
      * @param id A `'blob:nodedata:...` URL string returned by a prior call to `URL.createObjectURL()`.
      */
     export function resolveObjectURL(id: string): Blob | undefined;
-    export { Buffer };
+    export { Buffer, BufferEncoding, BufferConstructor };
```

Comment:
FWIW, I'm more interested in making `BufferEncoding` accessible. I evaluated all the types within `@types/node` and it seems these are the only two global types that aren't accessible in Deno via an export from a `node:x` module.